src/app/components/link-custom/link-custom.tsx

```diff
@@ -0,0 +1,54 @@
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { LinkArrowIcon } from '@/icons';
+
+import styles from './link-custom.module.scss';
+
+type LinkCustomProps = React.DetailedHTMLProps<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  HTMLAnchorElement
+> & {
+  label: string;
+  href: string;
+  size?: 'small' | 'medium' | 'large';
```

Comment:
Why does it optional if you pass the default params?

Reply:
Otherwise I get an error if I call this component without passing the mandatory parameters explicitly